diff -urN pkgutils-5.40.6.orig/pkgmk.conf pkgutils-5.40.6/pkgmk.conf
--- pkgutils-5.40.6.orig/pkgmk.conf	2017-11-30 10:28:57.000000000 -0600
+++ pkgutils-5.40.6/pkgmk.conf	2017-12-23 10:36:29.769284193 -0600
@@ -8,21 +8,6 @@
 # export JOBS=$(nproc)
 # export MAKEFLAGS="-j $JOBS"
 
-case ${PKGMK_ARCH} in
-	"64"|"")
-		;;
-	"32")
-		export CFLAGS="${CFLAGS} -m32"
-		export CXXFLAGS="${CXXFLAGS} -m32"
-		export LDFLAGS="${LDFLAGS} -m32"
-		export PKG_CONFIG_LIBDIR="/usr/lib32/pkgconfig"
-		;;
-	*)
-		echo "Unknown architecture selected! Exiting."
-		exit 1
-		;;
-esac
-
 # PKGMK_SOURCE_MIRRORS=()
 # PKGMK_SOURCE_DIR="$PWD"
 # PKGMK_PACKAGE_DIR="$PWD"
diff -urN pkgutils-5.40.6.orig/pkgmk.in pkgutils-5.40.6/pkgmk.in
--- pkgutils-5.40.6.orig/pkgmk.in	2017-11-30 10:28:57.000000000 -0600
+++ pkgutils-5.40.6/pkgmk.in	2017-12-23 10:36:15.555307660 -0600
@@ -790,12 +790,6 @@
 }
 
 main() {
-	# multilib
-	PKGMK_ARCH=64
-	if [ -f ".32bit" ]; then
-		PKGMK_ARCH=32
-	fi
-
 	local FILE TARGET
 
 	parse_options "$@"
