diff -urN pkgutils-5.40.7.orig/pkgmk.conf pkgutils-5.40.7/pkgmk.conf
--- pkgutils-5.40.7.orig/pkgmk.conf	2018-02-14 10:17:57.000000000 -0600
+++ pkgutils-5.40.7/pkgmk.conf	2018-06-03 10:44:00.348437912 -0500
@@ -2,27 +2,12 @@
 # /etc/pkgmk.conf: pkgmk(8) configuration
 #
 
-export CFLAGS="-O2 -march=x86-64 -pipe"
+export CFLAGS="-O2 -march=i686 -pipe"
 export CXXFLAGS="${CFLAGS}"
 
 # export JOBS=$(nproc)
 # export MAKEFLAGS="-j $JOBS"
 
-case ${PKGMK_ARCH} in
-	"64"|"")
-		;;
-	"32")
-		export CFLAGS="${CFLAGS} -m32"
-		export CXXFLAGS="${CXXFLAGS} -m32"
-		export LDFLAGS="${LDFLAGS} -m32"
-		export PKG_CONFIG_LIBDIR="/usr/lib32/pkgconfig"
-		;;
-	*)
-		echo "Unknown architecture selected! Exiting."
-		exit 1
-		;;
-esac
-
 # PKGMK_SOURCE_MIRRORS=()
 # PKGMK_SOURCE_DIR="$PWD"
 # PKGMK_PACKAGE_DIR="$PWD"
diff -urN pkgutils-5.40.7.orig/pkgmk.in pkgutils-5.40.7/pkgmk.in
--- pkgutils-5.40.7.orig/pkgmk.in	2018-02-14 10:17:57.000000000 -0600
+++ pkgutils-5.40.7/pkgmk.in	2018-06-03 10:43:37.474162611 -0500
@@ -790,12 +790,6 @@
 }
 
 main() {
-	# multilib
-	PKGMK_ARCH=64
-	if [ -f ".32bit" ]; then
-		PKGMK_ARCH=32
-	fi
-
 	local FILE TARGET
 
 	parse_options "$@"
